### ISSUE TITLE
October stripped wrongly

### ISSUE THREAD

**Issue by msom** [lgiordani/punch#25]
The month October gets stripped from `10` to `1` when using in the format `MM`. Seems like an `lstrip` is needed instead of a `strip` in [strftime](https://github.com/lgiordani/punch/blob/master/punch/version_part.py#L43).

**Comment by lgiordani** [comment]
Hi! Thanks for the report, I'll work on it. May you please share your `punch_config.py` file? Thanks

**Comment by msom** [comment]
Sure!

```
__config_version__ = 1

GLOBALS = {
    'serializer': '{{year}}.{{month}}.{{build}}',
}

FILES = ['setup.py']

VERSION = [
    {
        'name': 'year',
        'type': 'date',
        'fmt': 'YYYY'
    },
    {
        'name': 'month',
        'type': 'date',
        'fmt': 'MM'
    },
    {
        'name': 'build',
        'type': 'integer',
        'start_value': 0
    }
]

VCS = {
    'name': 'git',
    'commit_message': "Release {{ new_version }}",
}

ACTIONS = {
    'build': {
        'type': 'conditional_reset',
        'field': 'build',
        'update_fields': ['year', 'month']
    }
}
```

**Comment by lgiordani** [comment]
Fixed in version 1.4.2. Thanks for spotting it and for the code hint!

**Comment by msom** [comment]
Great, thanks a lot!
